frontend/static/js/form.js

```diff
@@ -268,6 +268,22 @@ PreloadController.prototype = {
   }
 };
 
+function issuesForSubdomainRemoval(status) {
+  var parentDomain = status.preloadedDomain;
+  var parentIsTLD = parentDomain.indexOf('.') == -1;
+  var error = {};
+  if (parentIsTLD) {
+    error.summary = "Domain is registered under a preloaded TLD";
+    error.message = "The entire TLD " + parentDomain + " is preloaded for HSTS and individual domain names cannot be removed.";
+  } else {
+    error.summary = "Domain is subdomain of a preloaded domain";
+    error.message = "This domain is a subdomain domain of " + parentDomain +
+                    ", which is on the preload list. To remove the HSTS " +
+                    "policy for " + status.name + ", you must remove " +
```

Comment:
Perhaps ..."example.com would need to be removed from the preload list" to avoid implying that the subdomain owner has authority over the ancestor?